### DISCUSSION TITLE
[error] [auth] Extension activation failed: "Contact Support. You are currently logged in as XXXXX."

### DISCUSSION THREAD

**Discussion by ITerydh** [community/community#118164] [Copilot Conversations] [answered]
### Select Copilot ERROR

Question

### Body

I have passed the student education certification, but I encountered the following issues when using it：

- [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as XXXXX."

What is the reason for this？

**Comment by ayim** [comment, marked as answer]
We are unable to provide any additional information beyond what Support will be able to provide.
I recommend you click the "Contact Support" button and engage with the team there.
